packages/network/src/logic/Node.ts

```diff
@@ -192,6 +192,7 @@ export class Node extends EventEmitter {
         if (!this.streams.isSetUp(streamId)) {
             this.logger.trace('add %s to streams', streamId)
             this.streams.setUpStream(streamId)
+            this.maintainTrackerConnections()
```

Comment:
This is here so the node can immediately react when a new stream has been joined to (as opposed to waiting for next interval execution)